src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -24,42 +24,51 @@ public GivenThatWeWantToUseAnalyzers(ITestOutputHelper log) : base(log)
         }
 
         [Theory]
-        [InlineData("WebApp", false)]
-        [InlineData("WebApp", true)]
-        [InlineData("WebApp", null)]
-        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, bool? isEnabled)
+        [MemberData(nameof(OffByDefaultGeneratorTestData))]
+        public void It_resolves_offbydefaultgenerator_correctly(GeneratorSpec generator, bool? isEnabled)
         {
             var asset = _testAssetsManager
-                .CopyTestAsset(testAssetName, identifier: isEnabled.ToString())
+                .CopyTestAsset("WebApp", identifier: isEnabled.ToString())
                 .WithSource()
                 .WithProjectChanges(project =>
                 {
                     if (isEnabled != null)
                     {
                         var ns = project.Root.Name.Namespace;
-                        project.Root.Add(new XElement(ns + "PropertyGroup", new XElement("EnableRequestDelegateGenerator", isEnabled)));
+                        // Delete when we have a new targeting pack that contains Microsoft.Extensions.Configuration.Binder.SourceGeneration.
+                        if (generator == GeneratorSpec.ConfigurationBinding)
```

Comment:
Is there a chance we can split up the unit test for the configuration generator and the request delegate one while this workaround is needed?

We can reintegrate them back into a single test theory once the targeting pack has flown through.